Formula/php.rb

```diff
@@ -0,0 +1,250 @@
+
+    # https://github.com/phpbrew/phpbrew/commit/18ef766d0e013ee87ac7d86e338ebec89fbeb445
+    # Unsure if this is still needed
+    inreplace "Makefile" do |s|
+      s.change_make_var! "EXTRA_LIBS", "\\1 -lstdc++"
+    end
+
+    system "make"
+    ENV.deparallelize
+    system "make", "install"
+
+    bin.install_symlink "phar.phar" => "phar"
+
+    config_path.install "./php.ini-development" => "php.ini"
+    (config_path/"php-fpm.d").install "sapi/fpm/www.conf"
+    config_path.install "sapi/fpm/php-fpm.conf"
+    inreplace config_path/"php-fpm.conf", /^;?daemonize\s*=.+$/, "daemonize = no"
+  end
+
+  def caveats
+    s = []
+
+    s << <<-EOS.undent
+      To enable PHP in Apache add the following to httpd.conf and restart Apache:
+          LoadModule php7_module #{opt_libexec}/apache2/libphp7.so
+
+          <FilesMatch \.php$>
+              SetHandler application/x-httpd-php
+          </FilesMatch>
+
+      Finally, check DirectoryIndex includes index.php
+          DirectoryIndex index.php index.html
+    EOS
+
+    s << <<-EOS.undent
+      The php.ini and php-fpm.ini file can be found in:
+          #{etc}/php/#{php_version}/
+    EOS
+
+    s.join "\n"
+  end
+
+  def post_install
+    (var/"log").mkpath
+    touch var/"log/php-fpm.log"
+
+    chmod 0755, lib/"php/.channels"
+    chmod 0755, lib/"php/.channels/.alias"
+    chmod 0644, (Dir.glob(lib/"php/.channels/**/*", File::FNM_DOTMATCH).reject { |a| a =~ %r{\/\.{1,2}$} || File.directory?(a) })
+
+    %w[
+      php/.depdblock
+      php/.filemap
+      php/.depdb
+      php/.lock
+    ].each do |f|
+      chmod 0644, lib/f
+    end
+
+    php_ini = "#{etc}/php/#{php_version}/php.ini"
+    system bin/"pear", "config-set", "php_ini", php_ini, "system"
+
+    %w[
+      ldap
+      mcrypt
+      tidy
+      imap
+      opcache
+    ].each do |e|
+      config_path = (etc/"php/#{php_version}/conf.d/ext-#{e}.ini")
+      extension_type = (e == "opcache") ? "zend_extension" : "extension"
+      if config_path.exist?
+        inreplace config_path, /#{extension_type}=.*$/, "#{extension_type}=#{e}.so"
+      else
+        config_path.write <<-EOS.undent
+          [#{e}]
+          #{extension_type}="#{e}.so"
+        EOS
+      end
+    end
+  end
+
+  def php_version
+    version.to_s.split(".")[0..1].join(".")
+  end
+
+  plist_options :startup => true, :manual => "php-fpm"
+
+  def plist; <<-EOPLIST.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <true/>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_sbin}/php-fpm</string>
+          <string>--nodaemonize</string>
+          <string>#{etc}/php/#{php_version}/php-fpm.conf</string>
```

Comment:
Missed removal of this line too with the `-y` option.

Reply:
thanks ^^